src/Resources/app/storefront/src/scss/category-filter.scss

```
@@ -1,5 +1,6 @@
 .filter-category-select-list {
-  width: 310px;
+  min-width:310px;
```

Please also run the IDE format here aswell.

```suggestion
  min-width: 310px;
```

---

This is still not resolved.